.github/workflows/dotnet.yml

```diff
@@ -46,28 +46,28 @@ jobs:
       shell: bash
       run: dotnet publish -c Release -r osx-arm64 --self-contained Refresher /p:Version=${VERSION}
 
-    - name: Remove duplicate .app for macOS x64
+    - name: Create macOS universal2 binary
       if: matrix.os == 'macos-latest'
       shell: bash
-      working-directory: Refresher/bin/Release/net8.0/osx-x64/publish/
-      run: rm -rfv ./*.app/Contents/MacOS/*.app && codesign -fs - --deep *.app
-
-    - name: Remove duplicate .app for macOS ARM64
-      if: matrix.os == 'macos-latest'
-      shell: bash
-      working-directory: Refresher/bin/Release/net8.0/osx-arm64/publish/
-      run: rm -rfv ./*.app/Contents/MacOS/*.app && codesign -fs - --deep *.app
```

Comment:
did you intend to remove the duplicate .app removal?

Reply:
Ah, no.